compiler/noirc_frontend/src/monomorphization/mod.rs

```diff
@@ -8,6 +8,46 @@
 //!
 //! The entry point to this pass is the `monomorphize` function which, starting from a given
 //! function, will monomorphize the entire reachable program.
+//!
+//! The monomorphized Ast (mAST) has a few notable differences from the Hir:
```

Comment:
```suggestion
//! The monomorphized AST (mAST) has a few notable differences from the HIR:
```
I feel like this is just more uniform with what we do in other comments. Both AST and HIR are acronyms as well.